pkg/api/resource/quantity.go

```diff
@@ -396,10 +397,16 @@ func (qf qFlag) Set(val string) error {
 	return nil
 }
 
+// Converts the value of the internal Quantity to a string. (used by flag & pflag)
 func (qf qFlag) String() string {
 	return qf.dest.String()
 }
 
+// States the type of flag this is (Quantity). (used by pflag)
+func (qf qFlag) Type() string {
+	return "Quantity"
```

Comment:
All of the other pflag values implement this with lower case.

I still don't know what/where this is used so it probably doesn't matter.